packages/alfa-rules/src/sia-r14/rule.ts

```diff
@@ -56,20 +56,20 @@ export default Rule.Atomic.of<Page, Element>({
 
       expectations(target) {
         // Removes all punctiation (underscore, hypen, brackets, quotation marks, etc)
-        function removePunctuation(input: string): string {
-          return input.replace(/\p{P}/gu, "");
+        // and normalise
+        function removePunctuationAndNormalise(input: string): string {
+          return normalize(input.replace(/\p{P}/gu, ""));
         }
 
-        const perceivableInnerTextFromElement = removePunctuation(
+        const perceivableInnerTextFromElement = removePunctuationAndNormalise(
           getPerceivableInnerTextFromElement(target, device)
         );
         const textContent = normalize(perceivableInnerTextFromElement);
         let name = "";
 
         const accessibleNameIncludesTextContent = test(
```

Comment:
And group lines logically.
```suggestion

        let name = "";
        const accessibleNameIncludesTextContent = test(
```